products/sle/main.pm

```diff
@@ -886,6 +886,10 @@ elsif (get_var('LIBSOLV_INSTALLCHECK')) {
 elsif (get_var("EXTRATEST")) {
     boot_hdd_image;
     load_extra_tests();
+    loadtest "console/coredump_collect";
```

Comment:
extratest infra is used by a lot of different projects currently , please next time when you plan to change flow like this for **everyone** inform more people. Quite unfortunate to find in wicked tests modules which I not confirm that they will be called :(
@Soulofdestiny , @rwx788 were you aware about this change ? is it ok for you ? 
at least I want to disabled for wicked , but let's clarify first what is more suitable - disable it for cases where maintainer is strongly against or just maybe enable only for cases it was expected to run. 
@baierjan what was you initial idea ? do flow did you have in mind ?

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/8410

Reply:
@asmorodskyi  why are you against coredump? This is standard debug information needed by developers, and the rest of is the correct shutdown of SUT

Reply:
I am not "against coredumps" , I am against breaking test isolation . Each test need to test something certain , it is easier to maintain and understand the results. 
Regarding "correct" shutdown - can you explain me why I should care about graceful shutdown of something what will be wiped out in a second after that ?